src/main/resources/db/changelog/changelog_beta.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+<changeSet author="Yevhenii" id="1">
+<createTable tableName="TVSeries">
+    <column autoIncrement="true" name="id" type="BIGINT">
+        <constraints primaryKey="true"/>
+    </column>
+    <column name="name" type="VARCHAR(50)">
+        <constraints nullable="false" unique = "true"/>
```

Comment:
why do you think it should be unique? Don't you think there is a change to get 2 same episode names for different tv shows?